blog/_posts/2021-07-06-distributed-rdataframe.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "RDataFrame is going distributed!"
+layout: archive
+author: Vincenzo Eduardo Padulano
+---
+
+[RDataFrame](https://root.cern/doc/master/classROOT_1_1RDataFrame.html) is ROOT's declarative interface for data
+analysis and since its release in 6.14 has seen tons of usages in real world analysis use cases and other examples in
+the wild. Parallelism has always been a staple of its design with the support of executing the event loop on all cores
+of a machine thanks to implicit multi-threading. Since ROOT 6.24, this aspect of RDataFrame has been enhanced further
+with distributed computing capabilities, allowing users to run their analysis on multi-node computing clusters through
+widely used frameworks such as [Apache Spark](https://spark.apache.org/), [Dask](https://dask.org/) or
+[AWS Lambda](https://aws.amazon.com/lambda/). The feature is currently in an experimental phase, try it out with this
+simple example (requires the python `pyspark` package):
+
+```python
+import ROOT
+ 
+# Point RDataFrame calls to the Spark specific RDataFrame
+RDataFrame = ROOT.RDF.Experimental.Distributed.Spark.RDataFrame
+ 
+# It still accepts the same constructor arguments as traditional RDataFrame
+df = RDataFrame("mytree", "myfile.root")
+ 
+# Continue the application with the traditional RDataFrame API
+sum = df.Filter("x > 10").Sum("y")
+h = df.Histo1D("x")
+ 
+print(sum.GetValue())
+h.Draw()
+```
+
+## Goals and implementation
+
+The main goal is to support distributed execution of any RDataFrame application. This has lead to the creation of a 
```

Comment:
```suggestion
The main goal is to support distributed execution of any RDataFrame application. This has led to the creation of a 
```

Reply:
I wanted to pour in some heavy metal :laughing: